app/src/main/res/layout/fragment_nearby_parent.xml

```diff
@@ -40,7 +40,24 @@
                 android:layout_height="match_parent"
                 android:background="@android:color/transparent" />
 
-        </FrameLayout>
+            <androidx.appcompat.widget.AppCompatTextView
+              android:id="@+id/tv_attribution"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_alignParentLeft="true"
+              android:layout_alignParentBottom="true"
+              android:layout_margin="8dp"
+              android:text="&#169;MapBox    &#169;OpenStreetMap \n Improve this map"
+              android:shadowColor="#000000"
+              android:shadowDx="2.5"
+              android:shadowDy="2.5"
+              android:shadowRadius="1.6"
+              android:textSize="14sp"
+              android:textColor="@android:color/white"
+              android:textAlignment="center"
+              android:textStyle="bold" />
```

Comment:
I took a stab at trying to make this look smaller and still legible in my small-screen device and the following is what helped/satisfied me:

```suggestion
             <androidx.appcompat.widget.AppCompatTextView
              android:id="@+id/tv_attribution"
              android:layout_width="wrap_content"
              android:layout_height="wrap_content"
              android:layout_alignParentLeft="true"
              android:layout_alignParentBottom="true"
              android:layout_margin="8dp"
              android:text="@string/map_attribution"
              android:textSize="10sp"
              android:textAlignment="center"
              android:textStyle="bold" />
```

With `map_attribution` defined to the following in `string.xml`:

```xml
<string name="map_attribution"><![CDATA[&#169; <a href="https://www.mapbox.com/about/maps/">Mapbox</a> &#169; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> <a href="https://www.mapbox.com/map-feedback/">Improve this map</a>]]></string>
```

Using CDATA, means we don't need `ClickableSpan`/`SpannableString` for the linking of the text. This has the advantage that the text clearly appears as a link and the code is readable.

Note: To ensure that's not translated we would've to define it in a separate file for strings that don't get translated.